src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -243,80 +285,84 @@ object TelemetryConstValues {
 
 object TelemetryQueryStatusFields {
   // Spark connector version
-  val SPARK_CONNECTOR_VERSION: String = "spark_connector_version"
+  val SPARK_CONNECTOR_VERSION = TelemetryFieldNames.SPARK_CONNECTOR_VERSION
   // The query operation: read vs write
-  val OPERATION: String = "operation"
+  val OPERATION = TelemetryFieldNames.OPERATION
   // Query statement
-  val QUERY: String = "query"
+  val QUERY = TelemetryFieldNames.QUERY
   // query ID if available
-  val QUERY_ID: String = "query_id"
+  val QUERY_ID = TelemetryFieldNames.QUERY_ID
```

Comment:
These seem redundant: why not just directly use `TelemetryFieldNames.XXX`?

Reply:
Do you mean we don't need to use TelemetryQueryStatusFields?

My idea is that we'd better to classify the field names for each telemetry message separately in one place. It is much clear for maintenance and query. As more telemetry messages are added in the future, this will be more important.